Cargo.toml

```diff
@@ -338,91 +160,91 @@ opt-level = "s"
 
 [[example]]
 name = "analog-stopwatch-with-spi-ssd1306"
-required-features = ["spi4", "tim2", "gpioe", "gpiog"] # stm32f429
+required-features = ["stm32f429"]
 
 [[example]]
 name = "blinky-timer-irq"
-required-features = ["tim2"] # stm32f411
+required-features = ["stm32f411"]
 
 [[example]]
 name = "can-send"
 required-features = ["can", "stm32f405"]
 
 [[example]]
 name = "delay-syst-blinky"
-required-features = ["device-selected"] # stm32f411
+required-features = ["stm32f411"]
 
 [[example]]
 name = "delay-timer-blinky"
-required-features = ["device-selected"] # stm32f411
+required-features = ["stm32f411"]
 
 [[example]]
 name = "display-touch"
 required-features = ["fsmc_lcd", "stm32f412"] #or stm32f413
 
 [[example]]
 name = "dwt-blinky"
-required-features = ["gpiog"] # "stm32f429"
+required-features = ["stm32f429"]
 
 [[example]]
 name = "dynamic-gpio"
-required-features = ["device-selected"]
+required-features = ["stm32f411"]
 
 [[example]]
 name = "f413disco-lcd-ferris"
-required-features = ["gpiod", "gpioe", "gpiof", "gpiog", "fsmc", "fsmc_lcd"] # stm32f413
+required-features = ["stm32f413", "fsmc_lcd"]
```

Comment:
I was kind of hoping we would go the other way around and make them less MCU model dependent? Not really a big fan of declaring that this only works on stm32f413... Same for the others of course.

Reply:
Good point. But I don't have good answer.
Do you have ideas?

Reply:
Not really, except that this is going in the wrong direction. Not quite sure how we can accomplish more genericity with the build.rs idea.